interfaces/console/src/app/console/subscribers/page.tsx

```diff
@@ -82,6 +82,15 @@ const Page = () => {
     },
   });
 
+  const [getDataUsages, { data: dataUsageData, loading: dataUsageLoading }] =
+    useGetDataUsagesLazyQuery({
+      pollInterval: 180000,
+      fetchPolicy: 'network-only',
```

Comment:
Polling for getDataUsages every 180 seconds might introduce unnecessary network overhead. Is there a specific reason for using this interval?

Reply:
Reason mentioned in PR description